VirtoCommerce.DynamicExpressionsModule.Data/Promotion/Rewards/RewardItemForEveryNumOtherItemInGetOfRel.cs

```diff
@@ -0,0 +1,31 @@
+using VirtoCommerce.Domain.Marketing.Model;
+
+namespace VirtoCommerce.DynamicExpressionsModule.Data.Promotion
+{
+    //For [] items of entry [] in every [] items of entry [] get [] % off no more than [] not to exceed $ []
+    public class RewardItemForEveryNumOtherItemInGetOfRel : RewardItemForEveryNumInGetOfRel
+    {
+        public ProductContainer ConditionalProduct { get; set; } = new ProductContainer();
+
+        #region IRewardExpression Members
+
+        public new PromotionReward[] GetRewards()
```

Comment:
Could you explain why do you use the `new` modificator here? It is absolutely  doesn't clear for me!

Reply:
It is a mistake in fact. Should be override instead. Will change